.github/workflows/ci.yml

```diff
@@ -389,8 +389,8 @@ jobs:
       - name: "Configure AWS credentials"
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-access-key-id: ${{ secrets.AWS_S3_ACSLICENSE_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_S3_ACSLICENSE_SECRET_ACCESS_KEY }}
```

Comment:
These credentials need to be added for DependaBot access as well, otherwise DependaBot PRs will fail.

Reply:
Thanks Domenico 🙏🏻 

<img width="766" alt="Screenshot 2023-02-28 at 09 46 39" src="https://user-images.githubusercontent.com/71768/221800727-4ab87490-9a70-414a-842b-5255c86b96d8.png">

Reply:
Thank **you!** 🙏